Module_Postman/activities/Act2_Creating_and_Testing_BitBloxs/7.md

```diff
@@ -0,0 +1,19 @@
+# API Authentication
+
+We have one last step to complete setup before we can start sending out requests. We still need an API key to access the API. 
+
+Go to the `Authorization` tab and select the authorization `type` as `API Key`. Then, set the value for `Key`, `Value`, `Add to` as the following. Again, we are using the environment variable that we set up earlier for our purpose. 
```

Comment:
As a general wee tip it's a good idea to include all required info in text and not be reliant on the content of images to carry out steps (like here where you need to look at the image to see what the values should be, esp the "add to" selection which I don't think has been covered elsewhere yet), makes it more accessible and helps if there's some problem loading images or whatever, also decouples the text from the images so e.g. you can update images without worrying about breaking the instructions..